pom.xml

```diff
@@ -315,7 +315,7 @@
         <mockito.version>3.0.0</mockito.version>
         <zookeeper.version>3.5.9</zookeeper.version>
         <jline.version>0.9.94</jline.version>
-        <hive.version>2.3.4</hive.version>
+        <hive.version>2.3.9</hive.version>
```

Comment:
looks like license update failed

Reply:
updated with` mvn license:aggregate-add-third-party@generate-and-check-licenses -Dlicense.skipAggregateAddThirdParty=false -B`

Reply:
fixed